### PR TITLE
[PWD-2517] Finalize Learning Material Page

### PR DESCRIPTION
**Pull request by hiljaker**
### What type of PR is this? (_check all applicable_)

- [ ] Feature
- [x] Bug Fix
- [ ] Documentation Update
- [ ] Setup
- [ ] Code Refactor
- [ ] Performance Improvements
- [ ] Test
- [ ] Build
- [ ] CI
- [ ] Revert

### CODE REVIEW

File: src/components/ProgramCard.tsx [hunk 1]
```diff
@@ -41,8 +41,8 @@ const ProgramTitle = styled(Typography)(() => ({
 
 const ProgramDescription = styled(Typography)(() => ({
   fontSize: '14px',
-  lineHeight: '20px',
-  minHeight: '60px',
+  lineHeight: '24px',
```
Comment (Vesals) [inline src/components/ProgramCard.tsx:44]: zak ini pake paragraph2 aja

File: src/components/ProgramCard.tsx [hunk 2]
```diff
@@ -17,20 +17,20 @@ const Card = styled(Box)(({ theme }) => ({
...
   backgroundImage: `linear-gradient(to right, ${theme.palette.neutral600.main} 50%, rgba(255,255,255,0) 0%)`,
   backgroundPosition: 'bottom',
-  backgroundSize: '14px 1px',
+  backgroundSize: '11px 1px',
   backgroundRepeat: 'repeat-x'
 }))
 
 const ProgramTitle = styled(Typography)(() => ({
-  minHeight: '49px',
+  minHeight: '52px',
```
Comment (Vesals) [inline src/components/ProgramCard.tsx:33]: ini nambah 3px karna ad yg gamuat?

Comment (hiljaker) [reply src/components/ProgramCard.tsx:33]: adjust lineheight adjust minheight juga, biar cardnya ga ada yg tinggi sebelah kalo ada judul yg cuma sebaris

File: src/components/ProgramCard.tsx (hunk 1 above)
Comment (hiljaker) [reply src/components/ProgramCard.tsx:44]: well mankk

File: src/components/ProgramCard.tsx (hunk 2 above)
Comment (Vesals) [reply src/components/ProgramCard.tsx:33]: oooh oke gw ikutin kalo gitu biar ga bentrok

**Review by Vesals — DISMISSED** [review]
🤖

**Review by Vesals — DISMISSED** [review]

**Review by Vesals — APPROVED** [review]

File: src/views/learning-material/components/ProgramCardList.tsx [hunk 3]
```diff
@@ -27,6 +28,16 @@ const ProgramCardList: FC<ProgramCardListProps> = ({
   initialLimit,
   ...props
 }) => {
+  const theme = useTheme()
+  const desktop = useMediaQuery(() => theme.breakpoints.up(1280))
```
Comment (kuntiarso) [inline src/views/learning-material/components/ProgramCardList.tsx:32]: 1280 pake xmd aja cuy

File: src/views/learning-material/components/ProgramCardList.tsx [hunk 4]
```diff
@@ -40,11 +51,11 @@ const ProgramCardList: FC<ProgramCardListProps> = ({
     <Box {...props}>
       {(fetchLoading && <LoadingScreen />) ||
         (!programs?.length && <EmptyProgram />) || (
-          <Grid container spacing="25.33px" mb={6}>
+          <Grid container columnSpacing="25.33px" rowSpacing="25px" mb={6}>
```
Comment (kuntiarso) [inline src/views/learning-material/components/ProgramCardList.tsx:54]: emang harus pake .33 kah?

File: src/views/learning-material/components/ProgramCardList.tsx (hunk 3 above)
Comment (hiljaker) [reply src/views/learning-material/components/ProgramCardList.tsx:32]: okee gan

File: src/views/learning-material/components/ProgramCardList.tsx (hunk 4 above)
Comment (hiljaker) [reply src/views/learning-material/components/ProgramCardList.tsx:54]: gua liat di desain ada .33 😆

File: src/views/learning-material/components/ProgramCardList.tsx (hunk 3 above)
Comment (Vesals) [reply src/views/learning-material/components/ProgramCardList.tsx:32]: eh xmd udah bisa???? nooooooooo gua keburu merge yg masi 1280

Comment (hiljaker) [reply src/views/learning-material/components/ProgramCardList.tsx:32]: gua sekalianin aja yaa phyn

**Review by Vesals — APPROVED** [review]

**Review by gangsararyop — APPROVED** [review]

**Pull request merged**